### DISCUSSION TITLE
When "Prefer metadata in original language" is on, make it so that BeatmapAttributeText in gameplay supports Unicode

### DISCUSSION THREAD

**Discussion by Johnny555GH** [ppy/osu#23340] [Ideas] [answered]
So, I have "Prefer metadata in original language" on in settings and it works fine in most areas, but in my skin, I added BeatmapAttributeText and it doesn't support Unicode at all, even with the Inter font. So, please make it so that BeatmapAttributeText supports Unicode just like in the SongSelect area.

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/pull/23369
